packages/ui/src/components/va-infinite-scroll/VaInfiniteScroll.vue

```diff
@@ -1,30 +1,26 @@
 <template>
   <component
-    :is="$props.tag"
+    :is="tag"
```

Comment:
```suggestion
    :is="$props.tag"
```

$props can help us separate props, computed and data properties in template. Do you see any reasons to remove this $props (Maybe some issues with types or smth like this)?

Reply:
returned everything back ))

Reply:
Just wanted to add that IDE highlights props \ data \ computed with a different color but yeah adding `props` may make it easier to review.